internal/engine/upper_test.go

```diff
@@ -691,101 +691,54 @@ k8s_resource('doggos', new_name='quux')  # rename "doggos" --> "quux"
 	f.assertAllBuildsConsumed()
 }
 
-func TestNoOpChangeToDockerfile(t *testing.T) {
+func TestConfigChange_NoOpChange(t *testing.T) {
```

Comment:
while I was here I cleaned up some old configChange tests to make them
a. more in line with how we currently test stuff
b. easier to read
c. not use fast build, since we'll deprecate it eventually